jenkins/gotest-dockerized.sh

```diff
@@ -36,7 +36,7 @@ mkdir -p "${HOST_ARTIFACTS_DIR}"
 # provided must be resolvable on the *HOST*, not the container.
 
 # default to go version 1.5 image tag
-KUBEKINS_TAG="1.5-v20161109-d9a98f2"
+KUBEKINS_TAG="1.5-v20161205-d664d14"
```

Comment:
maybe default to the 1.6 tag, but special-case 1.5 on the 1.5 tag, like we do for 1.2-1.4?

might want to turn the conditional on L41-45 into a `case`.